[ci] Permission denied github action bot

```bash
/home/runner/work/_temp/a4cb53ba-c6c0-4[25](https://github.com/layer5io/sistent/actions/runs/6431600817/job/17464936063#step:5:26)7-b302-e7d88755f4ed.sh: line 4: [: =: unary operator expected
lerna notice cli v7.3.0
lerna info versioning independent
lerna info ci enabled
lerna info Assuming all packages changed
lerna info getChangelogConfig Successfully resolved preset "conventional-changelog-angular"

Changes:
 - @layer5/sistent-components: 0.3.0 => 0.4.0
lerna info auto-confirmed 
lerna info execute Skipping releases
 - @layer5/rtk-query-codegen: 0.0.1 => 0.1.0
 - @layer5/sistent-svg: 0.6.6 => 0.7.0

lerna info git Pushing tags...
lerna ERR! Error: Command failed with exit code 128: git push --follow-tags --no-verify --atomic origin master
lerna ERR! remote: Permission to layer5io/sistent.git denied to github-actions[bot].
lerna ERR! fatal: unable to access 'https://github.com/layer5io/sistent/': The requested URL returned error: 403
lerna ERR!     at makeError (/home/runner/work/sistent/sistent/node_modules/execa/lib/error.js:60:11)
lerna ERR!     at handlePromise (/home/runner/work/sistent/sistent/node_modules/execa/index.js:118:[26](https://github.com/layer5io/sistent/actions/runs/6431600817/job/17464936063#step:5:27))
lerna ERR! lerna Command failed with exit code 1[28](https://github.com/layer5io/sistent/actions/runs/6431600817/job/17464936063#step:5:29): git push --follow-tags --no-verify --atomic origin master
lerna ERR! lerna remote: Permission to layer5io/sistent.git denied to github-actions[bot].
lerna ERR! lerna fatal: unable to access 'https://github.com/layer5io/sistent/': The requested URL returned error: [40](https://github.com/layer5io/sistent/actions/runs/6431600817/job/17464936063#step:5:41)3
Error: Process completed with exit code 128.
```

---

#### Contributor [Guides](https://docs.meshery.io/project/contributing) and Resources

- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/build-and-release)
- 📚 [Instructions for contributing to documentation](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)
  - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🎨 Wireframes and [designs for Layer5 site](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Layer5 Discussion Forum](https://discuss.layer5.io) and [Layer5 Community Slack](http://slack.layer5.io)

--- a/.github/workflows/release.yml
+++ b/.github/workflows/release.yml
@@ -35,7 +35,7 @@ jobs:
     needs: build
     runs-on: ubuntu-latest
     permissions:
-      contents: read
+      contents: write
       packages: write
     steps:
       - name: Checkout Repo

